tasmota/tasmota_xsns_sensor/xsns_109_sgp4x.ino

```diff
@@ -76,22 +82,41 @@ void sgp4x_Init(void)
   uint16_t error;
 
   sgp4x.begin(Wire);
-
   error = sgp4x.getSerialNumber(serialNumber, serialNumberSize);
 
   if (error) {
+    AddLog(LOG_LEVEL_ERROR, PSTR("SGP4X error: error during getserialnumber"));
```

Comment:
Please use `LOG_LEVEL_INFO`, this is still a minor error for the overall Tasmota.

Reply:
Fixed!